modules/passkey/docker/bundler/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/library/node:18
```

Comment:
You can `ln-s` this file, so it is always the same as the 4337 file. It is a lightweight way to share the docker file between both projects :shrug: